.github/workflows/ci.yml

```diff
@@ -6,6 +6,7 @@ jobs:
     build-mlibc:
         strategy:
             matrix:
+                arch: [x86_64, riscv64]
```

Comment:
Why not aarch64 too?

Reply:
We don't have Linux sysdeps for AArch64 yet